python/feathub/feature_tables/sinks/redis_sink.py

```diff
@@ -60,6 +61,8 @@ def __init__(
                          If not explicitly specified, the key would be a combination of
                          the namespace, all key field values, and the name of the
                          feature.
+        :param merge_map: If true, map-typed data would be merged into existing maps
```

Comment:
How about naming it `enable_hash_partial_update` so that it is closer to the partial update concept in SQL?